woff1/tools/validator/validator.py

```diff
@@ -2511,14 +2511,14 @@ def finishHTML(writer):
     writer.endtag("html")
     # get the text
     text = "<!DOCTYPE html PUBLIC \"-//W3C//DTD XHTML 1.0 Transitional//EN\" \"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd\">\n"
-    text += writer.compile()
+    text += writer.compile().decode()
```

Comment:
Binary data needs to be decoded in order to be able to perform string concatenation in Python 3.